Makefile

```diff
@@ -458,7 +458,7 @@ push-to-npm:
 ifeq ($(simulate), true)
 	@echo not publishing - simulation mode
 else
-	(cd $(pub-dir) ; npm publish)
+	(cd $(pub-dir) ; npm publish --access public)
```

Comment:
https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#publishing-scoped-public-packages